usaspending_api/recipient/management/sql/recipient_lookup/030_sam_parents.sql

```diff
@@ -10,7 +10,7 @@ INSERT INTO public.temporary_restock_recipient_lookup (
 )
 SELECT
   DISTINCT ON (ultimate_parent_unique_ide, ultimate_parent_legal_enti)
```

Comment:
Can you also add a bug fix for the `010 Adding Recipient records from SAM` step?

In that .sql script, instead of: 
```
SELECT
  DISTINCT ON (awardee_or_recipient_uniqu, legal_business_name)
```

It should be

```
SELECT
  DISTINCT ON (awardee_or_recipient_uniqu, UPPER(legal_business_name))
```

If not done it will collect the duns number more than once with different casings of the legal_business_name.